### DISCUSSION TITLE
option to set filename of output, defaulting to what quarto uses

### DISCUSSION THREAD

**Discussion by friendly** [nmfs-opensci/quarto_titlepages#53] [Q&A]
`quarto_titlepages` looks good, but it interferes with my workflow, because it uses the `title` of the book as the filename for output, rather than the default `index.tex`, `index.pdf`, ... used by quarto. Messes up Makefiles, ....

This happened after I had installed the extension, but did not use it in my book project.

I'm going to delete the extension until this is sorted out.

**Comment by eeholmes** [comment]
This is not from the extension, per se, but because this is a Quarto book project. For Quarto book project, by default it uses `title` yaml for the file names for the PDFs but you can change this by adding `output-file` to the yaml like so. Note, this isn't specific to the quarto_titlepages extension. Any Quarto book project has the same behavior.

```
book:
  title: "Here is the really long title that I don't want to use as a file name."
  output-file: "index"
```
